commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/config/DefaultedBukkitConfig.java

```diff
@@ -0,0 +1,115 @@
+package dev.jorel.commandapi.config;
+
+import org.jetbrains.annotations.ApiStatus;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Default config values for the plugin's config.yml file
+ */
+@SuppressWarnings("ClassEscapesDefinedScope")
+@ApiStatus.Internal
+public class DefaultedBukkitConfig extends DefaultedConfig {
+
+	public static final CommentedConfigOption<Boolean> USE_LATEST_NMS_VERSION = new CommentedConfigOption<>(
+		new String[] {
+			"Use latest version (default: false)",
+			"If \"true\", the CommandAPI will use the latest available NMS implementation",
+			"when the CommandAPI is used. This avoids all checks to see if the latest NMS",
+			"implementation is actually compatible with the current Minecraft version."
+		}, false
+	);
+
+	public static final CommentedConfigOption<Boolean> BE_LENIENT_FOR_MINOR_VERSIONS = new CommentedConfigOption<>(
+		new String[] {
+			"Be lenient with version checks when loading for new minor Minecraft versions (default: false)",
+			"If \"true\", the CommandAPI loads NMS implementations for potentially unsupported Minecraft versions.",
+			"For example, this setting may allow updating from 1.21.1 to 1.21.2 as only the minor version is changing",
+			"but will not allow an update from 1.21.2 to 1.22.",
+			"Keep in mind that implementations may vary and actually updating the CommandAPI might be necessary."
+		}, false
+	);
+
+	public static final CommentedConfigOption<Boolean> SHOULD_HOOK_PAPER_RELOAD = new CommentedConfigOption<>(
+		new String[] {
+			"Hook into Paper's ServerResourcesReloadedEvent (default: true)",
+			"If \"true\", and the CommandAPI detects it is running on a Paper server, it will",
+			"hook into Paper's ServerResourcesReloadedEvent to detect when /minecraft:reload is run.",
+			"This allows the CommandAPI to automatically call its custom datapack-reloading",
+			"function which allows CommandAPI commands to be used in datapacks.",
+			"If you set this to false, CommandAPI commands may not work inside datapacks after",
+			"reloading datapacks."
+		}, false
```

Comment:
We are saying that this defaults to true. Then why is the value here set to false?